react-list.es6

```diff
@@ -126,6 +136,7 @@ module.exports = class ReactList extends Component {
     window.removeEventListener('resize', this.updateFrame);
     this.scrollParent.removeEventListener('scroll', this.updateFrame, PASSIVE);
     this.scrollParent.removeEventListener('mousewheel', NOOP, PASSIVE);
+    clearTimeout(this.updateFrameTimeoutId);
```

Comment:
Checking the timeout is superfluous (see [MDN note](https://developer.mozilla.org/en-US/docs/Web/API/WindowOrWorkerGlobalScope/clearTimeout)).